src/xclim/indicators/atmos/_conversion.py

```diff
@@ -270,6 +271,26 @@ def cfcheck(self, **das) -> None:
     compute=indices.specific_humidity_from_dewpoint,
 )
 
+
+vapor_pressure_deficit = Converter(
+    title="Water vapour pressure deficit",
+    identifier="e_deficit",
```

Comment:
```suggestion
    identifier="vapor_pressure_deficit",
```

I'm assuming it's twice the same error, or is "e" short for "vapor_pressure" (climate noob here)